packages/wrangler/src/dev.tsx

```diff
@@ -444,6 +452,7 @@ export async function startDev(args: StartDevOptions) {
 					firstPartyWorker={configParam.first_party_worker}
 					sendMetrics={configParam.send_metrics}
 					testScheduled={args["test-scheduled"]}
+					experimentalMiniflare3={args.experimentalLocal}
```

Comment:
This feels like it should probably be called `experimentalLocal` instead of `experimentalMiniflare3`